README.md

```diff
@@ -1 +1,31 @@
 # bedrock-ledger-test-cli
+
+## Setup:
+
+  - You need to have a [web ledger node](https://github.com/digitalbazaar/bedrock-ledger-app-template/tree/initial) already running somewhere.
+  - You then use that node as the --hostName or -h to the cli 
+  - *note: do not specify a protocol such as https for the ledger node*
+
+```
+git clone git@github.com:digitalbazaar/bedrock-ledger-app-template.git
+cd bedrock-ledger-app-template
+npm i
+npm start //note the address of the node
+cd ..
+cd bedrock-ledger-test-cli
+npm run send -- -h your-ledger-node:port -k
+```
+
+cli commands
+```
+  .option('-h, --hostname <value>', 'The ledger host.')
+  .option('-k, --ignoreSslErrors', 'Use strict SSL.')
+  .option('-s, --send', 'Send a demonstration operation.')
+  .option('-r, --recordPath <value>', 'A path to a json record to be sent.')
```

Comment:
I guess this needs to be updated to match the options/descriptions that are now in the code.

Reply:
updated.